chainerx_cc/chainerx/native/native_device/batch_norm.cc

```diff
@@ -0,0 +1,12 @@
+#include "chainerx/native/op_regist.h"
+#include "chainerx/routines/normalization.h"
+
+namespace chainerx {
+namespace native {
+
+CHAINERX_REGISTER_OP_NATIVE(BatchNormForwardOp, GenericBatchNormForwardOp);
+CHAINERX_REGISTER_OP_NATIVE(BatchNormBackwardOp, GenericBatchNormBackwardOp);
+CHAINERX_REGISTER_OP_NATIVE(FixedBatchNormForwardOp, GenericFixedBatchNormForwardOp);
```

Comment:
Maybe we should introduce some mechanism in the future to register generic/fallback ops so that we won't have to explicitly register these generic ops for each and every backend.

Reply:
How about doing this in this PR?
I think that's essential (i.e. one of the goals) for op registry.

Reply:
Discussed offline that it's a bit involved and that it can be addressed in a separate PR. Need to discuss when and how however. cc. @beam2d.